hack/make/.build-rpm/docker-engine.spec

```diff
@@ -84,9 +84,12 @@ Requires: device-mapper >= 1.02.90-2
 %if 0%{?fedora} >= 22
 %global selinux_policyver 3.13.1-128
 %endif # fedora 22
-%if 0%{?centos} >= 7 || 0%{?rhel} >= 7 || 0%{?oraclelinux} >= 7
+%if 0%{?centos} >= 7 || 0%{?rhel} >= 7
 %global selinux_policyver 3.13.1-23
-%endif # centos,oraclelinux 7
+%endif # centos,rhel 7
+%if 0%{?oraclelinux} >= 7
+%global selinux_policyver 3.13.1-102.0.3.el7_3.15
```

Comment:
Hm, I assume you are installing docker-engine RPM that has OL selinux-policy dependency on version >= 3.13.1-102.0.3.el7_3.15.
That said in the current mainline code, the file `hack/make/.build-rpm/docker-engine-selinux.spec` has only the following global selinux version entry:
```
 12 URL: https://dockerproject.org
 13 Vendor: Docker
 14 Packager: Docker <support@docker.com>
 15 
 16 %global selinux_policyver 3.13.1-102
 17 %global selinuxtype targeted
 18 %global moduletype  services
 19 %global modulenames docker
 20 
 21 Requires(post): selinux-policy-base >= %{selinux_policyver}, selinux-policy-targeted >= %{selinux_policyver}, policycoreutils, policycoreutils-python libselinux-utils

```
not sure do you know why ?